crates/autopilot/src/run.rs

```diff
@@ -173,17 +173,9 @@ pub async fn run(args: Arguments, shutdown_controller: ShutdownController) {
         .await
         .unwrap();
 
-    let db_read = if let Some(db_read_url) = args.db_read_url
-        && args.db_write_url != db_read_url
-    {
-        Postgres::new(db_read_url.as_str(), args.insert_batch_size)
-            .await
-            .expect("failed to create read replica database")
-    } else {
-        db_write.clone()
-    };
```

Comment:
If we are removing the read replica functionality from autopilot (since it was pretty much just for the db metrics), let's also remove the command line arguments for it. It can be done by changing the deployment (to not configure --db-read-url) and then deploying this PR I presume.

The rolling update strategy has to be taken into consideration.

Reply:
> The rolling update strategy has to be taken into consideration.

Could you elaborate how/why?